docs/guide/built-in-functions.md

```diff
@@ -68,34 +68,34 @@ Total number of functions: **{{ $page.functionsCount }}**
+| DATEVALUE        | Parses a date string and returns it as the number of full days since [nullDate](../api/interfaces/configparams.md#nulldate).                                                                                                                                                                                                                                                                                                    | DATEVALUE(Datestring)                               |
+| DAY              | Returns the day of the given date value.                                                                                                                                                                                                                                                                                                                                                                                        | DAY(Number)                                         |
+| DAYS             | Calculates the difference between two date values.                                                                                                                                                                                                                                                                                                                                                                              | DAYS(Date2, Date1)                                  |
+| DAYS360          | Calculates the difference between two date values in days, in 360-day basis.                                                                                                                                                                                                                                                                                                                                                    | DAYS360(Date2, Date1[, Format])                     |
+| EDATE            | Shifts the given startdate by given number of months and returns it as the number of full days since [nullDate](../api/interfaces/configparams.md#nulldate).<br>Return type: `CellValueDetailedType.NUMBER_DATE` (non-compliant with the [OpenDocument](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html) standard)                                                | EDATE(Startdate, Months)                            |
+| EOMONTH          | Returns the date of the last day of a month which falls months away from the start date. Returns the value in the form of number of full days since [nullDate](../api/interfaces/configparams.md#nulldate).<br>Return type: `CellValueDetailedType.NUMBER_DATE` (non-compliant with the [OpenDocument](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html) standard) | EOMONTH(Startdate, Months)                          |
```

Comment:
I think the presented note is too horrifying.

WDYT about keeping the long explanation on the other page, and replacing the text here with a simple link:

> [See compatibility notes](list-of-differences.md)

Or even a footnote.

---

I think that, in fact, we could add this for *all* or *none* of the functions listed here that have an entry on the "List of differences" page.

Reply:
The rendering of the duplicate footnote looks weird to me (`[1:1]`), but I guess this is some convention that I was not aware of. It is certainly better.